rbd-target-api.py

```diff
@@ -568,12 +568,9 @@ def gateway(target_iqn=None, gateway_name=None):
     gateway_ip_list = target_config.get('ip_list', [])
 
     gateway_ip_list.append(ip_address)
-    first_gateway = (len(gateway_ip_list) == 1)
 
-    if first_gateway:
-        gateways = ['localhost']
```

Comment:
Nit: tons of calls rely on ```localhost``` so perhaps keep that check here (since if it's not listening on ```localhost```, things will break later)?

Reply:
Done